init.sh

```diff
@@ -24,7 +24,7 @@ export OCRD_LOGVIEW__PORT=$MONITOR_PORT_LOG
 export OCRD_CONTROLLER__JOB_DIR=/run/lock/ocrd.jobs
 export OCRD_CONTROLLER__HOST=$CONTROLLER_HOST
 export OCRD_CONTROLLER__PORT=$CONTROLLER_PORT
-export OCRD_CONTROLLER__USER=ocrd
+export OCRD_CONTROLLER__USER=admin
```

Comment:
Why are we changing the user here? Shouldn't we introduce a separate user admin `OCRD_CONTROLLER__ADMIN` variable here?

Reply:
Because `ocrd` was wrong to begin with (see initial implementation). It's the user that is backed by a worker semaphore on the Controller. Defies the logic of logging into the Controller to look at the job.

No need for a variable, everything is fixed – see startup.sh.